src/handlers/localstorage/defiPulse.ts

```diff
@@ -0,0 +1,2 @@
+// Key used for loading the cache with data from storage
+export const DEFI_PULSE_FROM_STORAGE = 'defiPulseFromStorage';
```

Comment:
We only used this for saving to storage and loading from storage - otherwise, the react-query logic should work as expected and we don't need to directly read from the cache, so this can be deleted and in useDPI hook you can just return data!